### ISSUE TITLE
Identify renamed lemmas

### ISSUE THREAD

**Issue by urkud** [alexjbest/leaff#5]
If a theorem disappeared and a new of the same type appeared, then show as "`abc` renamed to `cde`".

**Comment by alexjbest** [comment]
This should already happen, do you have an explicit example where it doesn't?

**Comment by urkud** [comment]
You're right, it identifies renames. However, in a more difficult case (renamed, restored the old lemma as a deprecated alias) it randomly reports new or old lemma as added.
